.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: '[Bug]: '
+labels: ''
+assignees: ''
+
+---
+
+<!--
+Thanks for opening a bug report!
+Before hitting the button, please fill in as much of the template below as you can.
+When describing the steps to reproduce the bug,
+try following your own instructions from scratch and make sure everything works as
+you expect.
+Be ready for follow-up questions, and please respond in a timely manner.
+If we can't reproduce a bug we might close your issue.
+-->
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. 
+2. 
+3. 
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Actual behavior**
+A clear and concise description of what actually happened.
+
+**Environment:**
+ - Kadalu Version: [e.g., 0.8.6]
+ - K8S_DIST: [e.g., kubernetes, openshift]
```

Comment:
- just a memo, we better add support for getting the k8s_dist as well from version command.

Reply:
@leelavg `k8s_dist` is just an environment variable right?

Reply:
Or are you expecting a command to read the yaml file from cluster and from there check this env var value?

Reply:
Yes, that's the idea however in a running cluster I don't think we can find that directly as the value is interpolated by that time iirc.

Anyways, it's just a todo not related to this PR.